thoth/metrics_exporter/jobs.py

```diff
@@ -137,6 +143,32 @@ def get_configmaps_per_namespace_per_label():
             )
 
 
+def get_ceph_results_per_type():
+    """Get the total number of results in Ceph per type."""
+    stores = [
+        ["adviser", AdvisersResultsStore()],
+        ["package-extract", AnalysisResultsStore()],
+        ["inspection", InspectionResultsStore()],
+        ["provenance-check", ProvenanceResultsStore()],
+        ["solver", SolverResultsStore()],
+        ["dependency-monkey", DependencyMonkeyReportsStore()],
+    ]
+    for store in stores:
+        try:
+            store[1].connect()
+            all_document_ids = store[1].get_document_listing()
+            list_ids = [str(cid) for cid in all_document_ids]
+            metrics.ceph_results_number.labels("bucket", store[0])
```

Comment:
What about changing this to result type? The bucket is always same:

```python
_MONITPRED_STORES = (
        AdvisersResultsStore(),
        AnalysisResultsStore(),
        InspectionResultsStore(),
        ProvenanceResultsStore(),
        SolverResultsStore(),
        DependencyMonkeyReportsStore(),
    )
...

def get_ceph_results_per_type():
    """Get the total number of results in Ceph per type."""
    for store in _MONITORED_STORES:
        try:
            if not store.is_connected():
                store.connect()
            all_document_ids = store[1].get_document_listing()
            list_ids = [str(cid) for cid in all_document_ids]
            metrics.ceph_results_number.labels("result_type", store.RESULT_TYPE)
```

According to:

https://github.com/thoth-station/storages/blob/7b4b6f10c2858b7f1d33bad084faaab358019f7c/thoth/storages/result_base.py#L32-L33

Also see connection handling to avoid multiple session instantiation.